shared/Microsoft.Azure.Devices.Shared/Twin.cs

```diff
@@ -32,6 +32,16 @@ public Twin(string deviceId) : this()
             DeviceId = deviceId;
         }
 
+        /// <summary>
+        /// Creates an instance of <see cref="Twin"/>
+        /// </summary>
+        /// <param name="twinProperties"></param>
+        public Twin(TwinProperties twinProperties)
```

Comment:
Just an FYI: This is adding new public API which means we'll need to increase the minor version for Shared.
I will address this in my next commit, prior to release.

Reply:
Thanks!